tests/014-predis-support.phpt

```diff
@@ -0,0 +1,75 @@
+--TEST--
+Predis userland functions are supported
+--SKIPIF--
+<?php
+if (!extension_loaded("scoutapm")) die("skip scoutapm extension required.");
+if (shell_exec("which composer") === null) die("skip composer not found in path.");
```

Comment:
Todo - find a way to make `SKIP` results return nonzero (to fail CI - don't really want to skip stuff in CI.

Reply:
Added a conditional in `if (!getenv('CI')) {` which does the trick. For that `skip` only check if not in CI basically.